arch/arm64/boot/dts/broadcom/bcm2837-rpi-3-b.dts

```diff
@@ -22,3 +22,14 @@
 &uart1 {
 	status = "okay";
 };
+
```

Comment:
This is also an upstream file. Have you considered using bcm2710-rpi-3-b.dts instead?